framework/email/tasks.py

```diff
@@ -68,35 +72,61 @@ def send_email(
             login=login,
             username=username,
             password=password,
+            reply_to=reply_to,
+            cc_addr=cc_addr,
         )
 
 
-def _send_with_smtp(from_addr, to_addr, subject, message, ttls=True, login=True, username=None, password=None):
+def _send_with_smtp(
+        from_addr,
+        to_addr,
+        subject,
+        message,
+        ttls=True,
+        login=True,
+        username=None,
+        password=None,
+        cc_addr=None,
+        reply_to=None,
+):
     username = username or settings.MAIL_USERNAME
     password = password or settings.MAIL_PASSWORD
 
     if login and (username is None or password is None):
         logger.error('Mail username and password not set; skipping send.')
-        return
+        return False
 
-    msg = MIMEText(message, 'html', _charset='utf-8')
+    msg = MIMEText(
+        message,
+        'html',
+        _charset='utf-8',
+    )
     msg['Subject'] = subject
     msg['From'] = from_addr
     msg['To'] = to_addr
 
-    s = smtplib.SMTP(settings.MAIL_SERVER)
-    s.ehlo()
-    if ttls:
-        s.starttls()
-        s.ehlo()
-    if login:
-        s.login(username, password)
-    s.sendmail(
-        from_addr=from_addr,
-        to_addrs=[to_addr],
-        msg=msg.as_string(),
-    )
-    s.quit()
+    if cc_addr:
+        msg['Cc'] = ', '.join(cc_addr)
```

Comment:
Check with product: How do they want the `To` and `CC` handled to avoid giving someone's email address away? The way this appears to be implemented, the recipients will both see the other's email address. In the case of a `cc:`, it might be that you want to `bcc:` both recipients.

Reply:
Yeah Product agrees BCC is better.